src/cmd/singularity/cli/build.go

```diff
@@ -38,6 +39,7 @@ func init() {
 	BuildCmd.Flags().BoolVarP(&force, "force", "f", false, "Delete and overwrite an image if it currently exists")
 	BuildCmd.Flags().BoolVarP(&noTest, "notest", "T", false, "Bootstrap without running tests in %test section")
 	BuildCmd.Flags().BoolVarP(&remote, "remote", "r", false, "Build image remotely")
+	BuildCmd.Flags().BoolVarP(&detached, "detached", "d", false, "Submit build job and print nuild ID (no stdout)")
```

Comment:
typo print **b**uild ID 

Also `(no stdout)` is a bit technical for users. Something like `no real-time logs` or similar?